pkg/controller/migmigration/restore.go

```diff
@@ -86,6 +86,13 @@ func (t *Task) ensureStageRestore() (*velero.Restore, error) {
 		return nil, liberr.Wrap(err)
 	}
 	newRestore.Labels[StageRestoreLabel] = t.UID()
+
+	stagePodImage, err := t.getDestStagePodImage()
+	if err != nil {
+		return nil, liberr.Wrap(err)
+	}
+	newRestore.Annotations[StageRestorePodImage] = stagePodImage
```

Comment:
@sseago please verify that I'm using the correct stage pod image here (dest cluster)